README.md

```diff
@@ -25,7 +25,7 @@ export default function start() {
 
   app.register(ReduxToken, Redux);
   app.register(ReducerToken, reducer);
-  app.register(EnhancerToken, reduxActionEnhancerFactory(EventEmitter));
+  app.register(EnhancerToken, reduxActionEmitterEnhancerFactory(UniversalEvents));
```

Comment:
I think we'll need to convert this package into a plugin due to the dependency on UniversalEvents

Reply:
I'll work on migrating this over to be a plugin.